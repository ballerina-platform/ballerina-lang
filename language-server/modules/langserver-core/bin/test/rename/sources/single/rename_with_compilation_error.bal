type TypeMap map<>;

public function main() {
    TypeMap tt = {};
}
