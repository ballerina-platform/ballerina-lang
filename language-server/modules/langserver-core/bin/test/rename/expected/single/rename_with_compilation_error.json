{
  "source": {
    "file": "rename_with_compilation_error.bal"
  },
  "position": {
    "line": 3,
    "character": 8
  },
  "prepareRename": {
    "valid": true
  },
  "result": {
    "changes": {
      "rename_with_compilation_error.bal": [
        {
          "range": {
            "start": {
              "line":0,
              "character":5
            },
            "end": {
              "line":0,
              "character":12
            }
          },
          "newText":"NewTypeMap"
        },
        {
          "range": {
            "start": {
              "line":3,
              "character":4
            },
            "end": {
              "line":3,
              "character":11
            }
          },
          "newText":"NewTypeMap"
        }
      ]
    }
  } 
}
